app/assets/stylesheets/views/_covid.scss

```diff
@@ -72,40 +72,6 @@ $c19-landing-page-header-background: #272828;
   }
 }
 
-.covid__instruction-list {
```

Comment:
Thanks for removing these leftovers after changing the approach in https://github.com/alphagov/collections/pull/2206!